.travis.yml

```diff
@@ -1,22 +1,30 @@
 language: go
 
 go:
-  # this is a lie - we will use 1.7 which isn't supported yet
-  # by downloading it later in "before_install" section
-  - 1.6
+  - 1.7
 
 env:
   - "MY_GOOS=linux MY_GOARCH=amd64"
-  - "MY_GOOS=darwin MY_GOARCH=amd64"
   - "MY_GOOS=windows MY_GOARCH=amd64"
```

Comment:
The reason we had `MY_*` style parameters was because the system go version was interfering with the downloaded version. Now that we are not downloading go - I'd say we should go back to using `GOOS` and `GOARCH` directly rather than the `MY_*` variants.